docs/troubleshooting-guide/docker.md

```diff
@@ -88,7 +88,7 @@ You can list the environment variables for a container with this command:
 The logs for the Docker container return an error message after the “Verifying Database Connection” line.
 
 #### How to fix this:
-Try to connect with `mysql` or `psql` commands with the connection string parameters you are passing in via the environment variables: https://metabase.com/docs/latest/operations-guide/start.html#configuring-the-metabase-application-database
+Try to connect with `mysql` or `psql` commands with the connection string parameters you are passing in via the environment variables. [https://metabase.com/docs/latest/operations-guide/start.html#configuring-the-metabase-application-database](Click here to learn more.)
```

Comment:
This link is backwards. Should be:

`[Click here to learn more](https://metabase.com/docs/latest/operations-guide/start.html#configuring-the-metabase-application-database)`